src/main/java/com/redhat/red/build/koji/KojiClient.java

```diff
@@ -325,11 +341,122 @@ public int getApiVersion()
         }
     }
 
+    public KojiSessionInfo krbLogin()
```

Comment:
I am fine with this. So, in the end there would be no references to the `kerberos` package in `KojiClient`, only references to this `KrbAuthenticator`.  There is a `org.apache.kerby.kerberos.kerb.auth.AuthContext` class which is meant to hold the state, but the kerby codebase currently has no references to it, and I didn't feel that it's worth the time to try to implement it. As I've said before, for a version 1.0.0, the kerby code seems very incomplete.